cheroot/test/test_conn.py

```diff
@@ -72,7 +68,7 @@ def err_before_read(req, resp):
 
     def one_megabyte_of_a(req, resp):
         """Render 1MB response."""
-        return ['a' * 1024] * 1024
+        return 'a' * 1024 * 1024
```

Comment:
Why did you change a series of chunks with one blob?

Reply:
I think that I did this because of the way that the  helper `Controller` is implemented https://github.com/cherrypy/cheroot/blob/master/cheroot/test/helper.py#L159-L169, it doesn't calculate the right size for the response if the handler didn't specified the `Content-Length` header. I remember that I was looking over the traffic with wireshark and saw that the response was there but the application only read the first 1024 bytes because of the `Content-Length` header, the rest was discarded. I'll double check then, I think my "fix" was only fixing the integration with testing code.